CacheSim/CacheSimCommon.inl

```diff
@@ -0,0 +1,628 @@
+#pragma once
+
+///////////
+// This file contains common code for all platform
+// It must be included exactly ONCE for the entire library
+//////////
+
+#include "Precompiled.h"
+
+#include "CacheSim.h"
+#include "CacheSimInternals.h"
+#include "CacheSimData.h"
+#include "GenericHashTable.h"
+#include "Md5.h"
+
+extern "C"
+{
+#include "udis86/udis86.h"
+}
+
+#define ARRAY_SIZE(a) (sizeof(a)/sizeof(a[0]))
+
+
+
+namespace CacheSim
+{
+    enum 
```

Comment:
Ugh, looks like I missed that file when I did my reformat pass to get things back to 2 spaces. Will fixing the indents resolve the issue or are there other subtle spacing issues that warrant just re-copying the code (like spaces inside parenthesis, etc.)?